src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -19,7 +19,7 @@ import io.iohk.ethereum.transactions.PendingTransactionsManager
 import scala.concurrent.Future
 import scala.util.{Failure, Success, Try}
 
-
+// scalastyle:off number.of.methods number.of.types
```

Comment:
We could, but as long as methods implementations are concise I don't see much value in doing so (other than make scalastyle happy).